native/android/build.gradle

```diff
@@ -13,7 +13,7 @@ buildscript {
         // Config for MapLibre
         rnmbglMapboxLibs = {
             implementation("org.maplibre.gl:android-sdk:9.2.1")
-            implementation("com.mapbox.mapboxsdk:mapbox-sdk-turf:5.3.0")
+            implementation ("com.mapbox.mapboxsdk:mapbox-sdk-turf:5.8.0")
```

Comment:
Maybe it is a dumb question, but why do we have to add these packages here, too? Or in general, when is it enough to add packages to package.json, and when do we have to add them in build.gradle / Podfile?

Reply:
Thats a good question. I think its like to work with platform specific peer dependencies.
Your react package f.e. @react-native-mapbox-gl/maps requires different peer deps depending on the platform you compile (defining in gradle or podsfile). But good question why this can not be auto generated somehow..